### ISSUE TITLE
ActiveStorage

### ISSUE THREAD

**Issue by santib** [rootstrap/rails_api_base#138]
I think it'd be a good idea to move from Carrierwave to ActiveStorage since it's the Rails way to manage files. I guess we can try it with one app before making the change here.

https://edgeguides.rubyonrails.org/active_storage_overview.html

Thoughts?

**Comment by matiasmansilla1989** [comment]
I agree with this idea but taking into consideration what @santiagovidal came up in Slack, we should support base 64 as well. Apparently, it's easy to do it manually so we can go ahead with this.
Great suggestion, thanks.
